--- config.def.h	2025-09-11 19:29:25.057471437 +0600
+++ config.h	2025-09-14 00:11:34.744082405 +0600
@@ -1,21 +1,23 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance */
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const unsigned int borderpx  = 3;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
+static const char *fonts[]          = { "Hack Nerd Font:size=15" };
+static const char dmenufont[]       = "Hack Nerd Font:size=15";
+static const char col_gray1[]       = "#141415";
 static const char col_gray2[]       = "#444444";
 static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
+static const char col_gray4[]       = "#cdcdcd";
 static const char col_cyan[]        = "#005577";
+static const char col_yellow[]      = "#e8b589";
+
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm] = { col_gray4, col_gray1,  col_gray2 },
+	[SchemeSel]  = { col_gray1, col_yellow, col_yellow  },
 };
 
 /* tagging */
@@ -34,7 +36,7 @@ static const Rule rules[] = {
 /* layout(s) */
 static const float mfact     = 0.55; /* factor of master area size [0.05..0.95] */
 static const int nmaster     = 1;    /* number of clients in master area */
-static const int resizehints = 1;    /* 1 means respect size hints in tiled resizals */
+static const int resizehints = 0;    /* 1 means respect size hints in tiled resizals */
 static const int lockfullscreen = 1; /* 1 will force focus on the fullscreen window */
 static const int refreshrate = 120;  /* refresh rate (per second) for client move/resize */
 
@@ -46,7 +48,7 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -58,23 +60,55 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_yellow, "-sf", col_gray4, NULL };
+static const char *termcmd[] = { "st", NULL };
+static const char *browsercmd[] = { "chromium", "--new-window", "--ozone-platform=x11", NULL };
+static const char *chatgptcmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://chatgpt.com/", "--profile-directory=Default", NULL };
+static const char *claudecmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://claude.ai/", "--profile-directory=Default", NULL };
+static const char *youtubecmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://youtube.com/", "--profile-directory=Default", NULL };
+static const char *whatsappcmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://web.whatsapp.com/", "--profile-directory=Default", NULL };
+static const char *calendarcmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://calendar.google.com/calendar", "--profile-directory=Profile 1", NULL };
+static const char *slackcmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://getmanzil.slack.com", "--profile-directory=Profile 1", NULL };
+static const char *discordcmd[] = { "chromium", "--new-window", "--ozone-platform=x11", "--app=https://discord.com/channels/@me", "--profile-directory=Profile 1", NULL };
+static const char *refdwmbar[] = { "setdwmbar.sh", NULL };
+static const char *signalappcmd[] = { "signal-desktop", NULL };
+static const char *screenshotregion[] = { "xscreenshot.sh", "region", NULL };
+static const char *screenshotwindow[] = { "xscreenshot.sh", "window", NULL };
+static const char *screenshotroot[] = { "xscreenshot.sh", "root", NULL };
+static const char *screenrecordregion[] = { "xscreenrecord.sh", "region", NULL };
+static const char *screenrecordroot[] = { "xscreenrecord.sh", "root", NULL };
+static const char *screenrecordrootaudio[] = { "xscreenrecord.sh", "root-audio", NULL };
 
 static const Key keys[] = {
 	/* modifier                     key        function        argument */
-	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
-	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
-	{ MODKEY,                       XK_b,      togglebar,      {0} },
+	{ MODKEY,                       XK_d,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY,                       XK_p,      togglebar,      {0} },
+	{ MODKEY,                       XK_b,      spawn,          {.v = browsercmd } },
+	{ MODKEY,                       XK_a,      spawn,          {.v = chatgptcmd } },
+	{ MODKEY|ShiftMask,             XK_a,      spawn,          {.v = claudecmd } },
+	{ MODKEY,                       XK_y,      spawn,          {.v = youtubecmd } },
+	{ MODKEY,                       XK_c,      spawn,          {.v = calendarcmd } },
+	{ MODKEY,                       XK_g,      spawn,          {.v = signalappcmd } },
+	{ MODKEY|Mod1Mask,              XK_g,      spawn,          {.v = slackcmd } },
+	{ MODKEY|ControlMask,           XK_g,      spawn,          {.v = discordcmd } },
+	{ MODKEY|ShiftMask,             XK_g,      spawn,          {.v = whatsappcmd } },
+	{ MODKEY,                       XK_r,      spawn,          {.v = refdwmbar } },
+	{ 0,                            XK_Print,  spawn,          {.v = screenshotregion } },
+	{ ShiftMask,                    XK_Print,  spawn,          {.v = screenshotwindow } },
+	{ MODKEY|ShiftMask,             XK_Print,  spawn,          {.v = screenshotroot } },
+	{ Mod1Mask,                     XK_Print,  spawn,          {.v = screenrecordregion } },
+	{ Mod1Mask|ShiftMask,           XK_Print,  spawn,          {.v = screenrecordroot } },
+	{ MODKEY|Mod1Mask,              XK_Print,  spawn,          {.v = screenrecordrootaudio } },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
 	{ MODKEY,                       XK_i,      incnmaster,     {.i = +1 } },
-	{ MODKEY,                       XK_d,      incnmaster,     {.i = -1 } },
+	{ MODKEY,                       XK_v,      incnmaster,     {.i = -1 } },
 	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} },
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
-	{ MODKEY,                       XK_Return, zoom,           {0} },
+	{ MODKEY|ShiftMask,             XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
-	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
+	{ MODKEY,                       XK_w,      killclient,     {0} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} },
 	{ MODKEY,                       XK_f,      setlayout,      {.v = &layouts[1]} },
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[2]} },
